update_ply.xsh

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env xonsh
-
-# update the ply repo bundled with xonsh
-git subtree pull --prefix xonsh/ply https://github.com/dabeaz/ply.git master --squash
```

Comment:
Yeah, and since it is for devs only, it is not that big of a deal. You might want to make sure that there are no lingering references in the devs guide